mat64/index_bound_checks.go

```diff
@@ -24,6 +25,7 @@ func (m *Dense) at(r, c int) float64 {
 	return m.mat.Data[r*m.mat.Stride+c]
 }
 
+// SetSym sets the element at row r, column c to the value v. It panics if c is not zero.
```

Comment:
This is a nice frankencopy message :). Function name from SymDense, behavior of Vector, and attached to Dense. I like it.